scripts/_terraform_util.sh

```diff
@@ -78,7 +86,15 @@ _terraform_cleanup() {
             export IS_CLEANUP_NEEDED="no"
             echo "Calling terraform destroy..."
             destroy_ret=0
-            terraform -chdir=/usr/primary destroy -input=false -auto-approve || destroy_ret=$?
+
+            # change terraform verbosity based on SHOW_PROXY_URL environment variable.
```

Comment:
It would be better to have a separate environment variable to control logging verbosity